File: read.rb
start_time = Time.now.to_f

if File.read("./lists.csv").include?("123.194.235.37")
    # do something
end

end_time = Time.now.to_f

puts "[ruby] File check finished in #{(end_time - start_time) * 1000} ms"

File: read.py
import time

start_time = time.time()
if "123.194.235.37" in open("./lists.csv").read():
    # do something
    pass
    
end_time = time.time()

print(f"[python] File check finished in {(end_time - start_time)*1000}ms")
